frontend/packages/design-system/src/tokens/fontConfig.ts

```diff
@@ -0,0 +1,11 @@
+export default {
+  Roboto: {
```

Comment:
I'd nest this under `fontFamily` in case we want to add any other tokens around typography (eg. font sizes)

Reply:
We may also have multiple font families

Reply:
> We may also have multiple font families
they could live on the same level as Roboto, couldn't they?

Reply:
Yeah other font families would, but not font sizes/letter spacings/weights/line heights. Not saying we need all of this, but it helps for it to be extensible.